bin/shell/osh.pl

```diff
@@ -1338,11 +1342,18 @@ sub main_exit {
         osh_debug("idle_timeout: finally using " . $idleTimeout{$timeout} . " for $timeout");
     }
 
+    # if $command matches this option, set stealth_stdout for ttyrec
+    my $stealth_stdout = 0;
+    if (my $ttyrecStealthStdoutPattern = OVH::Bastion::config("ttyrecStealthStdoutPattern")->value) {
```

Comment:
Shouldn't it be something like

```perl
    my $stealth_stdout = 0;
    if (my $ttyrecStealthStdoutPattern = OVH::Bastion::config("ttyrecStealthStdoutPattern")) {
        $stealth_stdout = $command =~ $ttyrecStealthStdoutPattern->value;
    }
```

? In case of _false_ result.

Reply:
Both work, but if your version is more readable, let's go for it